harness/determined/cli/job.py

```diff
@@ -100,6 +87,89 @@ def update(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def process_updates(args: Namespace) -> None:
+    session = setup_session(args)
+    for arg in args.operation:
+        inputs = validate_operation_args(arg)
+        _single_update(session=session, **inputs)
+
+
+def _single_update(
+    job_id: str,
+    session: Session,
+    priority: str = "",
+    weight: str = "",
+    resource_pool: str = "",
+    behind_of: str = "",
+    ahead_of: str = "",
+) -> None:
+    update = bindings.v1QueueControl(
+        jobId=job_id,
+        priority=int(priority) if priority != "" else None,
+        weight=int(weight) if weight != "" else None,
+        resourcePool=resource_pool if resource_pool != "" else None,
+        behindOf=behind_of if behind_of != "" else None,
+        aheadOf=ahead_of if ahead_of != "" else None,
+    )
+    bindings.post_UpdateJobQueue(session, body=bindings.v1UpdateJobQueueRequest([update]))
+    return
```

Comment:
redundant return

Reply:
....sorry I don't know how I didn't see this until AFTER merging. I'll fix this in a follow up PR.